### ISSUE TITLE
iphone X Statusbar bug with whatsapp and instagram 

### ISSUE THREAD

**Issue by Hazuum** [ryannair05/Little11#22]
black bar appears in instagram and bottom panel disappears in whatsapp

**Comment by luccasclezar** [comment]
Yep, when Bottom Inset is toggled, WhatsApp is not showing the TabBar and the bottom of the app is not limited by the screen, with the messages list going off-screen.
